app/src/modules/sparql.js

```diff
@@ -17,24 +20,22 @@ async function querySparql (query, endpoint = SPARQL_ENDPOINT) {
     .catch((err) => console.log(err))
 }
 
-export { querySparql }
-
-export function parseSPARQL (response) {
+function parseSparql (response) {
```

Comment:
maybe an `if` in case the response.results.bindings is empty. I remember that when I was working with the endpoint I will always handle a response with a 200 but with an empty body if no information is found. 

```js
if (!response.results.bindings.length) return null
```

Reply:
I would think this should not be implemented here. When you receive a response, you can check response length and handle it appropriately inside the component. How each component handle a null result/response might differ